core/src/neighbourhood/Neighbourhood.ts

```diff
@@ -12,6 +12,9 @@ export class Neighbourhood {
     @Field()
     meta: Perspective
 
+    @Field()
+    author: string;
+
```

Comment:
This is redundant since the Neighbourhood exists as expression in the Neighbourhood Language. But I think I understand why you thought it's needed: Perspective.ts only inlcudes a `Neighbourhood` property and not a `NeighbourhoodExpression`. Two ways to solve this without introducing this redundancy:
1. Change `Perspective.neighbourhood` to be a `NeighbourhoodExpression` including the author (and signature!)
2. We have `Perspective.sharedUrl` which is the URL of the `NeighbourhoodExpression`. We could get it from there. Actually `Perspective.neighbourhood` is just a cached version of that without the expression fields. 

So I think better would be option 1, which means changing the initialisation when publishing and joining Neighbourhoods in `Ad4mCore.neighbourhoodPublishFromPerspective()` and `Ad4mCore.installNeighbourhood()`